space-cli/cmd/modules/remote-services/commands.go

```diff
@@ -10,8 +10,10 @@ import (
 func GenerateSubCommands() []*cobra.Command {
 
 	var generateService = &cobra.Command{
-		Use:  "remote-services",
-		RunE: actionGenerateService,
+		Use:     "remote-service [path to config file]",
```

Comment:
@sharadregoti @YourTechBud Can you check this part again and confirm the usage? I commit changes proposed by @sharadregoti

Reply:
Seems to be correct. @sharadregoti?

Reply:
yes